.circleci/config.yml

```diff
@@ -272,6 +315,10 @@ workflows:
           image_tag: heroku/pack:20-build
           image_tag_alias: heroku/pack:20-build
           requires:
+            - test-go-20
+            - test-java-20
+            - test-php-20
```

Comment:
Once the Ruby getting-started-guide addition is made above, Ruby will need adding here

Reply:
fixed in 643440a